cfme/common/provider.py

```diff
@@ -756,7 +756,7 @@ def _fill_credential(form, cred, validate=None):
                 'token_verify_secret': cred.verify_token,
                 'validate_btn': validate
             })
-            if validate:
+            if validate_hawkular:
```

Comment:
@pavelzag, could you please show me where you want to use this new option ?
btw, could you please explain what is the reason for it ?